app/addons/dashboard/stores.js

```diff
@@ -0,0 +1,169 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+define([
+  'app',
+  'api',
+  'addons/activetasks/actiontypes'
+], function (app, FauxtonAPI, ActionTypes) {
+
+  var DashboardStore = FauxtonAPI.Store.extend({
+
+    init: function (collectionTable, backboneCollection) {
+      this._prevSortbyHeader = 'started_on';
+      this._headerIsAscending = true;
+      this._selectedRadio = 'All Tasks';
+      this._sortByHeader = 'started_on';
+      this._searchTerm = '';
+      this._collection = collectionTable;
+      this._pollingIntervalSeconds = 5;
+      this.sortCollectionByColumnHeader(this._sortByHeader);
+      this._backboneCollection = backboneCollection;
+    },
+
+    getSelectedRadio: function () {
+      return this._selectedRadio;
+    },
+
+    setSelectedRadio: function (selectedRadio) {
+      this._selectedRadio = selectedRadio;
+    },
+
+    getPollingInterval: function () {
+      return this._pollingIntervalSeconds;
+    },
+
+    setPollingInterval: function (pollingInterval) {
+      this._pollingIntervalSeconds = pollingInterval;
+    },
+
+    setPolling: function () {
+      clearInterval(this.getIntervalID());
+      var id = setInterval(function () {
+        this._backboneCollection.pollingFetch();
+        this.setCollection(this._backboneCollection.table);
+        this.sortCollectionByColumnHeader(this._prevSortbyHeader, false);
+        this.triggerChange();
+      }.bind(this), this.getPollingInterval() * 1000);
+
+      this.setIntervalID(id);
```

Comment:
Meh! I'd drop this method, drop `id` and just set `this._intervalID` to the result of the setInterval method (unless you think it's needed for tests?)